common/fc_cmdhelp.cpp

```diff
@@ -113,9 +93,8 @@ void cmdhelp_display(struct cmdhelp *pcmdhelp, bool sort, bool gui_options,
   fc_fprintf(stderr, _("Usage: %s [option ...]\nValid option are:\n"),
              pcmdhelp->cmdname);
 
-  cmdarg_list_sort(pcmdhelp->cmdarglist, cmdarg_compare);
-  cmdarg_list_iterate(pcmdhelp->cmdarglist, pcmdarg)
-  {
+  std::sort(pcmdhelp->cmdarglist->begin(), pcmdhelp->cmdarglist->end());
```

Comment:
This `sort` will sort by the value of the pointer, while entries were sorted in alphabetical order before.

Reply:
Hehe, I was checking it on server and results were the same, but it seems only client uses it